src/Hyperion/Analysis.hs

```diff
@@ -49,11 +49,13 @@ namesOf = go []
     coerce = contramap (const ()) . fmap (const ())
 
 analyze
-  :: Text -- ^ Benchmark name
-  -> Sample -- ^ Measurements
+  :: Text -- ^ Executable name.
+  -> Text -- ^ Benchmarks name.
+  -> Text -- ^ Benchmark name.
+  -> Sample -- ^ Measurements.
   -> Report
-analyze name samp = Report
-    { _reportBenchName = name
+analyze executableName benchesName name samp = Report
+    { _reportBenchName = Text.concat [executableName, ".", benchesName, ".", name]
```

Comment:
I repeat my comment for a previous version of this: why are you prefixing with the executable name? What is "benchmarks name"?